docs/src/components/components/panel/panel.html

```diff
@@ -0,0 +1,22 @@
+<section class="panel">
+  <header class="panel-header">
+    <h2 class="panel-title">Panel title</h2>
+    <div class="panel-header-content">
+      <span>Panel data point</span>
+      <button>Panel action</button>
+    </div>
+  </header>
+
+  <section class="panel-documentation panel-documentation-desc">
+    <p>Use this marketplace to create service instances for
+      spaces in this org. Then bind service instances to apps using the command line.
+      <a href="https://docs.cloud.gov/apps/managed-services/">Learn about using
+      service instances and marketplaces</a>.
+    </p>
+  </section>
```

Comment:
I feel like in fractal terms, `panel-documentation` is a component and with some tweaking, could be used with the `{% render %}` syntax here.